raiden/ui/cli.py

```diff
@@ -150,12 +150,14 @@ def options(func):
             show_default=True,
         ),
         option(
-            '--network-type',
+            '--environment',
             help=(
-                'Specify the network type (main or test).\n'
+                'Specify the environment (production or development).\n'
+                'The "production" setting adds some safety measures and is mainly intended '
+                'for running Raiden on the mainnet.\n'
             ),
-            type=click.Choice(['main', 'test']),
-            default='test',
+            type=click.Choice(['production', 'development']),
```

Comment:
Shouldn't we try using the enum value here? (I don't know if it's possible, just asking)

Reply:
I don't see why not, will change it...